sinks/api/decoder.go

```diff
@@ -120,9 +125,9 @@ func (self *decoder) getContainerMetrics(container *cache.ContainerElement, labe
 				Name:   supported.Name,
 				Labels: labelsAsString,
 			}
-			// TODO: remove this once the heapster source is tested to not provide duplicate metric.Statss.
 
-			if data, ok := self.lastExported[key]; ok && data.After(metric.Stats.Timestamp) {
+			// TODO: remove this once the heapster source is tested to not provide duplicate metric.Stats.
```

Comment:
Just noticed this @piosz. The last time I tested, I did not see any duplicate stats. We can add a Vlog here to validate my claim.